command/agent/command.go

```diff
@@ -1081,8 +1081,12 @@ func (c *Command) handleReload(config *Config) (*Config, error) {
 		errs = multierror.Append(errs, fmt.Errorf("Failed unloading checks: %s", err))
 		return nil, errs
 	}
+	if err := c.agent.unloadMetadata(); err != nil {
```

Comment:
Right now this just resets the metadata map so it can't actually get an error; i'll change it to not return anything for simplicity